kubemarine/plugins/__init__.py

```diff
@@ -522,8 +523,33 @@ def apply_expect(cluster, config, plugin_name=None):
 # **** PYTHON ****
 
 def verify_python(cluster, step):
-    # TODO: verify fields types and contents
-    return
+    module_path, _ = utils.determine_resource_absolute_file(step['module'])
+    method_name = step['method']
+    method_arguments = step.get('arguments', {})
+
+    # Try to import the module
+    try:
+        spec = importlib.util.spec_from_file_location('module', module_path)
+        module = importlib.util.module_from_spec(spec)
+        spec.loader.exec_module(module)
```

Comment:
Now the module is executed twice that might be not expected. I suggest to ensure to execute the module only once.

Reply:
This is still executed twice, when called from apply_python and verify_python. I understand that we already do that for our own modules https://github.com/Netcracker/KubeMarine/blob/0.18.0/kubemarine/resources/configurations/defaults.yaml#L542-L547, but this is becoming worse for custom modules if they are used only once. If it is hard to fix, we can introduce restriction to documentation that execution should be idempotent, duplicates of static variables and classes might appear and so on.

Reply:
I tried skipping exec_module for verify python, but it was giving error method not found so without getting a method reference it will fail to perform validation of method signature.